src/main/resources/intentionDescriptions/MakeEncoderIntention/after.elm.template

```diff
@@ -0,0 +1,3 @@
+encode : Int -> Encode.Value
```

Comment:
Just like the decode example, I would try to make it a little more exciting by showing how it works with a record.